slate/source/localizable/index.html.md

```diff
@@ -525,3 +525,11 @@ Access the ALA   <a href="https://events-test.ala.org.au/" target="_blank">Event
 For full api documentation see <a href="./openapi/index.html?urls.primaryName=events">Open API specification</a>
 
 &nbsp;
+
+## 17. Ecodata
+
+Services for the <a href="https://fieldcapture-test.ala.org.au/" target="_blank">MERIT</a> and <a ref="https://biocollect-test.ala.org.au/acsa/" target="_blank">BioCollect</a> applications.
```

Comment:
Since this link is just a reference to the parent app and it's not reading form config - I've normally set this value to prod on other sections. This won't really be changed often so I'd recommended setting it to prod link so that it doesn't have to be updated when were merge to master for `prod` release.